src/common/reducers/handlers/cards.js

```diff
@@ -145,4 +121,25 @@ function createCardFromProps(props) {
   return card;
 }
 
+// Saves a card, either as a new card or replacing an existing card.
+function saveCard(state, card) {
+  // Is there already a card with the same ID (i.e. we're currently editing it)
+  // or that is identical to the saved card (i.e. we're replacing it with a card with the same name)?
+  const existingCard = state.cards.find(c => c.id === card.id || areIdenticalCards(c, card));
```

Comment:
It's not a problem because `id` is randomly generated for imported cards (see `cardsFromJson()`.